sample-app/index.js

```diff
@@ -56,7 +57,8 @@ mongoStorage.connect(
         scheduler.install(app)
         fb.installWebhook(app, config.facebook.webhook_path, config.facebook.webhook_verification_token, botEngine, scheduler)
         tg.installWebhook(app, config.telegram.webhook_host, config.telegram.webhook_path, botEngine, scheduler)
-
+        storage.createIndex()
```

Comment:
The name for this method is very uninformative: what index, why should it be created? 
Also, why here and not before the webhooks are installed?